flake8_django/checkers/model_meta.py

```diff
@@ -0,0 +1,97 @@
+import ast
+
+from flake8_django.checkers.base_model_checker import BaseModelChecker
+from flake8_django.checkers.issue import Issue
+
+
+class DJ09(Issue):
+    code = 'DJ09'
+    description = 'Model: {model} must define class Meta'
+
+
+class DJ10(Issue):
+    code = 'DJ10'
+    description = 'Model: {model} has no verbose_name'
+
+
+class DJ11(Issue):
+    code = 'DJ11'
+    description = 'Model: {model} has no verbose_name_plural'
+
+
+class ModelMetaChecker(BaseModelChecker):
+    model_name_lookup = 'Model'
+
+    def checker_applies(self, node):
+        for base in node.bases:
+            if self.is_model_name_lookup(base) or self.is_models_name_lookup_attribute(base):
+                return True
+        return False
+
+    @staticmethod
+    def has_meta_class(element):
+        # for node in element.body[0].body:  # type: ignore
+        for node in element.body:  # type: ignore
+            if isinstance(node, ast.ClassDef):
+                # class Meta
+                if node.name == 'Meta':
+                    return node
+        return
+
+    def has_verbose_name(self, meta_class_node):
+        return self._has_element(meta_class_node, 'verbose_name')
+
+    def has_verbose_name_plural(self, meta_class_node):
+        return self._has_element(meta_class_node, 'verbose_name_plural')
+
+    @staticmethod
+    def _has_element(element, target_name):
+        for node in ast.iter_child_nodes(element):
+            if not isinstance(node, ast.Assign):
+                continue
+            attr = node.targets[0].id
```

Comment:
Could you add a check here for `isinstance(node.targets[0], ast.Name)`. This is to avoid the issue in which we could have something like this:

```python
class Meta:
    a, b = 1
```

Reply:
done

Reply:
Yes, you're right, when the first there is the tuple assign as you point it would fail. 
Added the theck.

Reply:
Done in tests/fixtures/model_with_meta.py and tests/test_model_meta.py